src/platform/plugins/shared/data/public/query/timefilter/use_timefilter.ts

```diff
@@ -114,7 +114,7 @@ export function createUseTimefilterHook(
     },
   });
 
-  const timeStateConsumer$ = timeState$.pipe(share());
+  const timeStateConsumer$ = timeState$.pipe(shareReplay(1));
```

Comment:
needs to be a shareReplay so I can fetch the latest value when triggering a new sample fetch without having to remember it in xstate